packages/venia-ui/lib/components/CartPage/PriceSummary/priceSummaryFragments.js

```diff
@@ -1,7 +1,7 @@
 import gql from 'graphql-tag';
 
 import { DiscountSummaryFragment } from './discountSummary';
-import { GiftCardSummaryFragment } from './giftCardSummary';
+import { GiftCardSummaryFragment } from './queries/giftCardSummary.js';
```

Comment:
`./queries/giftCardSummary.js` instead of `./queries/giftCardSummary` because we are only tapping on the resolution of js files in webpack.

Reply:
What does this mean?

Reply:
`./queries/giftCardSummary.js` does not exist. There is only `./queries/giftCardSummary.ce.js` or `./queries/giftCardSummary.ee.js`. But we still write `./queries/giftCardSummary.js` because the plugin is only looking for `js` file imports, once it comes across this import, it realizes that `ce` and `ee` versions of it are available and resolves to them based on the env variable.